src/features/Courses/TableLayout/index.jsx

```diff
@@ -11,11 +11,12 @@ const TableLayout = ({
   handleChangeSelectedCourses,
   isLoading,
   actionButton,
+  isSelectable,
 }) => (
   <Row className="justify-content-center my-4">
     <Col>
       <DataTable
-        isSelectable
+        isSelectable={isSelectable}
```

Comment:
If this is not a custom prop or you don't modify its value is better to pass just props: https://github.com/Pearson-Advance/frontend-app-institution-portal/blob/master/src/features/Main/Table/index.jsx#L20
so you don't have to define every prop